css/styleguide.scss

```diff
@@ -920,9 +909,10 @@ h3 + .button_wrapper {
   background-color: $color-gray-warm-light;
 }
 
-.usa-color-gray-cool-light {
-  background-color: $color-gray-cool-light;
-}
+// no such var
+//.usa-color-gray-cool-light {
+//  background-color: $color-gray-cool-light;
```

Comment:
This line was killing my build because `$color-gray-cool-light` didn't seem to be available. I don't know why, so I commented out the whole rule. Brute force, but I wasn't sure this was an important rule for us.

Reply:
Isn't it used on the color palette page or did it change?

Reply:
yeah, this broke that. I wonder why that var broke my build?

Reply:
Yeah needs to be added back in. See the screenshot from https://federalist-proxy.app.cloud.gov/preview/18f/web-design-standards-docs/dw-fix-roadmap-links/components/colors/#palette:
<img width="655" alt="screen shot 2018-01-19 at 11 32 41 am" src="https://user-images.githubusercontent.com/5249443/35168065-7bd1b834-fd0c-11e7-99a8-e44f7a8b32e3.png">

Reply:
Found it. I modified that var by mistake in my utilities branch in the main repo.
Fixing